Makefile.PL

```diff
@@ -203,6 +203,8 @@ sub install {
     my ($self) = @_;
     my $install = $self->SUPER::install(@_);
 
+    return $install if $OSNAME eq 'MSWin32';
```

Comment:
Can you explain why it is needed to shortcut install() function here for Appveyor ? I think this is not needed at all and may break stuff in some case, maybe under cygwin.